### DISCUSSION TITLE
Inquiry Regarding the Release Date of MedusaJS's Extensible Admin for Ecommerce Platform

### DISCUSSION THREAD

**Discussion by kelachim** [medusajs/medusa#3742] [Q&A]
Dear MedusaJS Team,

I hope this letter finds you well. My name is Michal Hrbáček, and I am a member of the ViceVerse team. We are currently considering using your ecommerce platform for an upcoming project. However, we have a pressing question that we would like to address with you.

As you may know, an extensible admin feature is an essential component for further development. We have been following your development closely and are eagerly awaiting the release of your extensible admin feature. Our project has a tight deadline, and we need to know when we can expect this feature to be available.

We understand that software development is a complex process that can take time, but we would appreciate any information you can provide regarding the release date of your extensible admin. This information will help us make an informed decision about whether we should start our project without it or wait for the update.

Thank you for your attention to this matter. We look forward to hearing from you soon and hope to continue our collaboration in the future.

Best regards,
Michal Hrbáček
ViceVerse Team

**Comment by olivermrbl** [comment]
Hi there @kelachim - sorry for the slow follow-up.

We are amid finalizing a PoC and an RFC fleshing out the technical design of the solution.

I expect us to publish an alpha version early next week to get some initial reactions to what we've built. Will keep you posted.

@kasperkristensen is leading these efforts, so he can answer any further questions.
